Makefile

```diff
@@ -165,10 +165,19 @@ taskmetadata-validator:
 ecr-execution-role-image:
 	$(MAKE) -C misc/ecr $(MFLAGS)
 
+.PHONY: gocyclo static-check
```

Comment:
Please either add these to the `.PHONY` on line 25 or do one each above the targets.

```make
.PHONY: gocyclo
gocyclo:

.PHONY: static-check
static-check: gocyclo
```

Reply:
added one per target.